app/core/config.py

```diff
@@ -39,27 +39,29 @@
 with open("/data/blast_config.json") as f:
     BLAST_CONFIG = json.load(f)
 
+
 # logging configuration
+
 LOGGING_LEVEL = logging.DEBUG if DEBUG else logging.INFO
 LOGGERS = ("uvicorn.asgi", "uvicorn.access")
 
-log = logging.getLogger("gunicorn.access")
-
 logging.getLogger().handlers = [InterceptHandler()]
 for logger_name in LOGGERS:
     logging_logger = logging.getLogger(logger_name)
     logging_logger.handlers = [InterceptHandler(level=LOGGING_LEVEL)]
 
 logger.configure(handlers=[{"sink": sys.stderr, "level": LOGGING_LEVEL}])
 
-logging.info(BLAST_CONFIG)
-
 # Nextflow Configurations
-NF_TOKEN = config("NF_TOKEN")
-NF_COMPUTE_ENV_ID = config("NF_COMPUTE_ENV_ID")
-NF_PIPELINE_URL = config("NF_PIPELINE_URL")
-NF_WORK_DIR = config("NF_WORK_DIR")
-SEQERA_API = config("SEQERA_API")
-NF_WORKSPACE_ID = config("NF_WORKSPACE_ID")
+NF_TOKEN = config("NF_TOKEN", default="")
+NF_COMPUTE_ENV_ID = config("NF_COMPUTE_ENV_ID", default="")
+NF_PIPELINE_URL = config("NF_PIPELINE_URL", default="")
+NF_WORK_DIR = config("NF_WORK_DIR", default="")
+SEQERA_API = config("SEQERA_API", default="")
+NF_WORKSPACE_ID = config("NF_WORKSPACE_ID", default="")
```

Comment:
Without all these variables VEP won't work. Is setting default to "" a good idea ?

Reply:
You're right. I'll move them to docker-compose.

Reply:
That will serve the purpose. Also as it is already set in the .env file. Exposing .env file to docker-compose would work for both docker based env/ virtualenv/production across alll the environment dev/prod etc.